app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -118,7 +118,9 @@ def bulk_resume
       end
       def index
```

Comment:
When you're doing changes to the index, wouldn't it make sense to use the [index layout](https://github.com/theforeman/foreman/blob/develop/app/views/api/v2/layouts/index_layout.json.erb)?

Reply:
I would be ok with leaving this for separate refactoring issue.